### DISCUSSION TITLE
Release proposal for the version 2.7.0

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#1921] [General]
Hi, community

It has been more than two months since the release(2.6.0) and many exciting features have been done in this period, especially most of JSON commands are supported(huge thanks to @PragmaTwice who drives this process). Now I would like to propose the next minor release version: 2.7.0, it will include all commits in the unstable branch until https://github.com/apache/kvrocks/commit/6cb5b3aa57805299359e73e903212b2e25dbfcad. 

The deadline for the cherry-picking period has been set as December 13, 2023, at 19:00 UTC+8. No new commits will be accepted after this time.

Any comments are welcome, looking forward to hearing from you!

@apache/kvrocks-committers 

## Cherry-pick commits

- https://github.com/apache/kvrocks/pull/1927
- https://github.com/apache/kvrocks/pull/1928

**Comment by PragmaTwice** [comment]
LGTM.

Thank you for driving this release!

**Reply by PragmaTwice** [reply]
One comment: seems the stream group commands are still unfinished / unusable, should we still try to disable them?

**Reply by git-hulk** [reply]
Oh, yes. Thanks for your reminder. I asked @Yangsx-1 about the progress a few days ago, the stream group commands will be done after 2.7.0, so we should opt-out of those unfinished commands.

**Reply by git-hulk** [reply]
Done by https://github.com/apache/kvrocks/commit/6d1571d797994ec2db3036d9cae052b7137f6650

**Comment by tisonkun** [comment]
> most of JSON commands are supported

What's the concrete status? Is it possible we support all the commands that _can_ be supported?

**Reply by tisonkun** [reply]
Because when we write the release note or blog, when we introduce this achievement to our users, we'd better have a list for supported commands and tell what will happen for those unsupported yet.

**Reply by git-hulk** [reply]
Sure, guess we can also publish a blog on the website to introduce what's new in the release.

**Reply by git-hulk** [reply]
> What's the concrete status? Is it possible we support all the commands that can be supported?

https://kvrocks.apache.org/docs/supported-commands#json-commands and yes all commands can be supported in Kvrocks.

**Comment by git-hulk** [comment]
The 2.7.0 cherry-pick window has been closed now, we will call the release vote soon.
